gaphor/UML/treemodel.py

```diff
@@ -31,6 +31,10 @@ def __init__(self, element: Base | None):
         self.element = element
         self.sync()
 
+    __gsignals__ = {
```

Comment:
Hi @mvinca, this is working great for me now. I think @amolenaar may be able to give more guidance, but if we add a watch at https://github.com/gaphor/gaphor/blob/7227e969326c1fb30b7685e282651736e722e9c4/gaphor/UML/actions/activity.py#L26 then we shouldn't have to create custom `__gsignals__` to update the tree when a parameter node is added.